notifier/main.ts

```diff
@@ -26,8 +26,12 @@ bot.chatType(["group", "supergroup"]).filter((ctx) =>
   ) {
     return;
   }
+  const entity = repliedMessage.entities?.[0];
```

Comment:
Confer https://deno.land/x/grammy@v1.14.0/mod.ts?s=Context&p=prototype.entities

Reply:
Like this?

```suggestion
  const entity = ctx.entities("text")?.[0];
```

Reply:
Yes, but without `?.`.

Reply:
Oh and also, the name of the entity isn't `"text"`. It's either `"url"` or `"text_link"`.